locales-pending/premium.ftl

```diff
@@ -298,6 +298,30 @@ data-broker-profiles-estimated-time = Est. time to complete: { $estimated_time }
 # $exposure_reduction is the percentage of exposures that are data brokers.
 data-broker-profiles-exposure-reduction = Exposure reduction: { $exposure_reduction }%
 
+# Welcome to Premium Data Broker Profiles
+
+# Exposure reduction chart
+# The number inside <nr> will be displayed in a large font,
+# the label inside <label_line_1> and <label_line_2> will be shown in a smaller font. <label_line_1> and <label_line_2> spread the label over two lines, ideally of similar length.
```

Comment:
You either make this a section comment, or you split it into two distinct comment.

Right now, this is tied only to the string following (-heading), and other string won't have any comment associated.

Reply:
(Section comments [are comments that start with `##`](https://projectfluent.org/fluent/guide/comments.html).)